Dockerfile

```diff
@@ -11,4 +11,8 @@ RUN GRADLE_USER_HOME=cache ./gradlew buildDeb -x test && \
   cd .. && \
   rm -rf workdir
 
+EXPOSE 8090
+ENV services.redis.baseUrl=redis://localhost:6379
+ENV redis.connection=redis://localhost:6379
```

Comment:
These env & port settings should either be defaults in the mounted configuration, or overridden in a kubernetes or docker compose config.

Reply:
It is a good practice to declare in Dockerfile all exposed PORTs and ENVs. To do that you have to define a default value, even one that will be overwritten by docker-compose.

Reply:
If those configuration settings are meant to be configured through Docker ENV parameters, you should list them in Dockerfile as a way for developers to know they exist!